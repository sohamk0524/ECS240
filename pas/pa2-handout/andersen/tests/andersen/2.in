c Test case with only address-of statements.
p 10 5
s 0 2 -1 2
s 0 4 -1 2
s 0 6 -1 8
s 0 8 -1 1
s 0 10 -1 10
