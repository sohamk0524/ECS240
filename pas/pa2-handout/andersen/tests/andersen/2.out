pt 10 10
pt 2 2
pt 4 2
pt 6 8
pt 8 1
